docs/blog/contributor-development-maintenance-guide.en-US.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Contributor development maintenance guide
+date: 2023-03-10
+author: kiner-tang
+---
+
+Hi, my name is [kiner-tang](https://github.com/kiner-tang). My personal work is closely related to Ant Design, so I can be considered a heavy user of Ant Design.It is for this reason that I am slowly working my way up from a user to a Contributor, contributing some of the problems I encountered and the new features I concluded to the community, and finally being honored to become a member of Ant Design Collaborator.
+
+During the journey from user to contributor, and then from contributor to collaborator, I also encountered many problems. I would like to take this opportunity to sort out and summarize these problems, hoping to be helpful to new contributors and collaborators joining Ant Design community.
```

Comment:
... I would like to share my experience with the problem I meet. Hope it can help you to join the Ant Design community.

Reply:
done